extensions/lifecycle/bucketProcessor/policy.json

```diff
@@ -13,7 +13,8 @@
                 "s3:GetObjectTagging",
                 "s3:GetObjectVersionTagging",
                 "s3:GetObject",
-                "s3:GetObjectVersion"
+                "s3:GetObjectVersion",
+                "s3:ReplicateObject"
```

Comment:
It seems weird that `ReplicateObject` permission is needed to update the metadata of the object, where all that bucketProcessor needs is to update a few states...
→ this should probably be reviewed eventually, to have separate routes and/or permissions for different fields (e.g. bucket processor should not be able to create a new version or modify the 'location' of the data, though it may modify some fields : cold storage status, ...) : can you please create a followup ticket about making this permissions cleaner & more granular?